code/graphics/gropenglpostprocessing.cpp

```diff
@@ -408,36 +404,14 @@ void gr_opengl_post_process_end()
 		}
 	}
 
-	// bloom uniforms, but only if we did the bloom
-	if (bloomed) {
```

Comment:
AppVeyor is complaining because this was the only usage of `bloomed` (set on line 357 above). If `opengl_post_pass_bloom()`'s return value is going to be ignored, maybe it shouldn't return anything?